packages/api/src/hooks/useOnfidoServiceToken.ts

```diff
@@ -0,0 +1,32 @@
+import { ALPHA_3_TO_ALPHA_2 } from '../constants';
+import useQuery from '../useQuery';
+import useSettings from './useSettings';
+
+// TODO: Merge service token hooks into a single service token hook call
```

Comment:
There are alot of similar calls for service tokens such as `useCtraderServiceToken` and `useDerivezServiceToken`, once this PR is merged I plan to refactor these hooks into a single hook `useServiceToken` if its a good idea